src/bundle/Resources/public/js/scripts/fieldType/ezauthor.js

```diff
@@ -0,0 +1,130 @@
+(function (global, doc) {
+    const SELECTOR_REMOVE_AUTHOR = '.ez-js-remove-author';
+    const SELECTOR_AUTHOR_ROW = '.ez-author-row';
+    const SELECTOR_FIELD = '.ez-field-edit-ezauthor';
+    const SELECTOR_FORM_LABEL = '.form-control-label';
+    const SELECTOR_FIELD_EMAIL = '.ez-sub-field-email';
+    const SELECTOR_FIELD_NAME = '.ez-sub-field-name';
+
+    class EzAuthor extends global.eZ.baseFieldValidator {
+        validateName(event) {
+            const isError = !event.target.value.trim() && event.target.required;
+            const fieldNode = event.target.closest(SELECTOR_FIELD_NAME);
+            const errorMessage = global.eZ.errors.emptyField.replace('{fieldName}', fieldNode.querySelector(SELECTOR_FORM_LABEL).innerHTML);
+
+            return {
+                isError: isError,
+                errorMessage: errorMessage,
+            };
+        }
+
+        validateEmail(event) {
+            const isEmpty =  !event.target.value.trim();
+            const isValid = global.eZ.errors.emailRegexp.test(event.target.value);
+            const isError = event.target.required && isEmpty || !isValid;
+            const fieldNode = event.target.closest(SELECTOR_FIELD_EMAIL);
+            let errorMessage;
+
+            if (isEmpty) {
+                errorMessage = global.eZ.errors.emptyField.replace('{fieldName}', fieldNode.querySelector(SELECTOR_FORM_LABEL).innerHTML);
+            } else if (!isValid) {
+                errorMessage = global.eZ.errors.invalidEmail;
+            }
+
+            return {
+                isError: isError,
+                errorMessage: errorMessage,
+            };
+        }
+
+        setIndex(parentNode, template) {return template.replace(/__index__/g, parentNode.querySelectorAll(SELECTOR_AUTHOR_ROW).length)}
+
+        updateDisabledState(parentNode) {
+            const isEnabled = parentNode.querySelectorAll(SELECTOR_AUTHOR_ROW).length > 1;
+
+            [...parentNode.querySelectorAll(SELECTOR_REMOVE_AUTHOR)].forEach(btn => {
+                if (isEnabled) {
+                    btn.removeAttribute('disabled');
+                } else {
+                    btn.setAttribute('disabled', true);
+                }
+            });
+        }
+
+        removeItem(event) {
+            const authorNode = event.target.closest(SELECTOR_FIELD);
+
+            event.target.closest(SELECTOR_AUTHOR_ROW).remove();
+
+            this.updateDisabledState(authorNode);
+        }
+
+        addItem(event) {
+            const authorNode = event.target.closest(SELECTOR_FIELD);
+            const template = authorNode.dataset.prototype;
```

Comment:
I know, we've talked about this. But it's so confusing ... this `prototype` thing